lanarky/callbacks/llm.py

```diff
@@ -14,6 +14,7 @@
 )
 
 SUPPORTED_CHAINS = ["LLMChain", "ConversationChain"]
+ANSWER_KEY = "answer"
```

Comment:
i think this should be a parameter for the callback in case someone uses a different output key.